deployment/ecr/distributed-load-testing-on-aws-load-tester/load-test.sh

```diff
@@ -33,6 +33,11 @@ if [ "$TEST_TYPE" != "simple" ]; then
     fi
 
     sed -i -e "s|$TEST_ID.jmx|$JMETER_SCRIPT|g" test.json
+
+    # copy bundled plugin jars to jmeter extenstion folder to make them available to jmeter
+    JMETER_EXT_PATH=`find ~/.bzt/jmeter-taurus -type d -name "ext"`
+    BUNDLED_PLUGIN_DIR=`find $PWD -type d -name "plugins"`
+    cp -v $BUNDLED_PLUGIN_DIR/*.jar $JMETER_EXT_PATH
```

Comment:
Can you add a failure check here? For example, if either `JMETER_EXT_PATH` or `BUNDLED_PLUGIN_DIR` don't exist, `cp` command would fail, so it would be safer to add the directory check.

Reply:
thanks good point - I added two checks: 

* Skip copying if no /plugins folder is present in the upload (`BUNDLED_PLUGIN_DIR` is empty).
* Fail fast if jmeter extension folder not found and plugins present (`JMETER_EXT_PATH` is empty).

wdyt ?

Reply:
@beomseoklee I believe I have all cases covered and the addition is backwards compatible (just skipping the step if no plugins bundled in upload zip).

I'll do my best to address further concerns as quickly as possible.
Would be awesome to get this in - my company currently runs a fork (where this change originates from) and I'd be happy to back to the upstream version, thanks! (:

Reply:
I think this has been addressed, please lmk if I'm missing something!